core/lib/multivm/src/tracers/call_tracer/mod.rs

```diff
@@ -38,4 +52,24 @@ impl CallTracer {
         let cell = self.result.as_ref();
         cell.set(result).unwrap();
     }
+
+    fn push_call_and_update_stats(&mut self, call: FarcallAndNearCallCount) {
+        self.stack.push(call.clone());
+
+        self.max_stack_depth = self
+            .max_stack_depth
+            .max(self.stack.len() + call.near_calls_after);
+        self.max_near_calls = self.max_near_calls.max(call.near_calls_after);
+    }
+
+    fn increase_last_and_update_stats(&mut self, value: usize) {
```

Comment:
- Nit: AFAICT, this is always called with `value == 1`. Could this be called with another `value`? If not, it may make sense to remove the arg in order to simplify the code.
- Bikeshedding: Maybe name this method something like `increase_near_call_count`? The current name looks a bit esoteric.